bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/core/TelldusCoreDeviceController.java

```diff
@@ -179,10 +182,12 @@ private void sendEvent(Device device, int resendCount, boolean isdimmer, Command
     }
 
     private void increaseDecrease(Device dev, IncreaseDecreaseType increaseDecreaseType) throws TellstickException {
-        String strValue = ((TellstickDevice) dev).getData();
         double value = 0;
-        if (strValue != null) {
-            value = Double.valueOf(strValue);
+        if (dev instanceof TellstickDevice device) {
+            String strValue = null;
```

Comment:
Is this assignment necessary?

Reply:
Otherwise I get a warning the car is not initialised

Reply:
Okay. Maybe a slight refactor will make it more readable?
```java
String strValue = device.getData();
if (strValue != null) {
    value = Double.valueOf(strValue);
}

Reply:
Way better. Wondering why i took that turn... :-)